pull_request_template.md

```diff
@@ -4,11 +4,11 @@ This pull request description is not the actual RFC. To view the actual RFC, cli
 

```

Comment:
⚠️    __potential-blocker:__ This file used to needto be in `.github/PULL_REQUEST_TEMPLATE.md`, I was checking the docs, and it sounds like github may of undid that requirement.  But wanted to surface it as a followup potentially.

Reply:
💭 __non-blocking:__ I'd also add back in your helpful comment

## Note to RFC reviewers

This pull request description is not the actual RFC. To view the actual RFC, click __Files Changed__ in the top menu for this pull request. 

__NOTE__: I recommend reading the files in their rendered format rather than their raw format.

Reply:
Proposed Markdown:

```markdown
## Note to RFC reviewers

This pull request description is not the actual RFC. To view the actual RFC, click __Files Changed__ in the top menu for this pull request. 

__NOTE__: I recommend reading the files in their rendered format rather than their raw format.
```

Reply:
➡️ I think I could get the Links to the actually viewable format embedded onto the RFC PR so folks could get to these more easily.